intel-sgx/pcs/src/lib.rs

```diff
@@ -160,6 +164,16 @@ fn intel_signature_deserializer<'de, D: Deserializer<'de>>(deserializer: D) -> R
     crate::get_ecdsa_sig_der(signature).map_err(|_| de::Error::custom("Failed ECDSA signature conversion"))
 }
 
+fn iso8601timestamp<'de, D: Deserializer<'de>>(deserializer: D) -> Result<DateTime<Utc>, D::Error> {
+    // Same as %Y-%m-%dT%H:%M:%S%.f%:z, i.e. 0, 3, 6 or 9 fractional digits for seconds and colons in the time zone offset.
+    // This format also supports having a Z or UTC in place of %:z. They are equivalent to +00:00.
+    // https://docs.rs/chrono/0.4.39/chrono/format/strftime/index.html#fn7
+    const ISO8601_FORMAT: &'static str = "%+";
```

Comment:
Not sure if having to explain this shorthand in a comment is better than just putting the full format string.